tests/publiccloud/img_proof.pm

```diff
@@ -71,7 +71,7 @@ sub run {
     parse_extra_log(IPA => $img_proof->{results});
     assert_script_run('rm -rf img_proof_results');
 
-    $instance->ssh_script_run(cmd => 'sudo chmod a+r /var/tmp/report.html', no_quote => 1);
+    $instance->ssh_script_run(cmd => 'sudo chmod a+r /var/tmp/report.html || true', no_quote => 1);
     $instance->upload_log('/var/tmp/report.html', failok => 1);
 
```

Comment:
this way you'll get error by uploading a not-existing file

Reply:
> this way you'll get error by uploading a not-existing file

No because `failok => 1` is set in the next line.

Reply:
ok

Reply:
then you can add that same to ssh_ command !

Reply:
> then you can add that same to ssh_ command !

That option is not supported by `ssh_script_run`.  We need to fix those when the time comes.